src/ashley/templates/partials/user_icon.html

```diff
@@ -0,0 +1,6 @@
+{% load i18n %}
+{% load custom_tags %}
+
+{% if topic|is_user_instructor:user %}
+    <i class="icon_writer fas fa-award" aria-hidden="true" title="{%trans 'Instructor'%}"></i>
```

Comment:
I'm not sure about accessibility. You're hiding it but it conveys some important information. WDYT @mbenadda?

Reply:
Following the instructions of font awesome I think we're good there, optionally they recommend to add a span to describe more, but in our case I think this is sufficient. Machina is not really good at it, for instance it as no title on href of the same block By: <a href="{{ poster_url }}">{{ username }}</a>

ref https://fontawesome.com/how-to-use/on-the-web/other-topics/accessibility

Web Fonts with CSS: Semantic Icons 

If your icons have semantic meaning, you’ll need to manually add a few things so that your icon is appropriately accessible:

aria-hidden="true" attribute.
Provide a text alternative inside a <span> (or similar) element. Also include appropriate CSS to visually hide the element while keeping it accessible to assisitive technologies.
title attribute on the icon to provide a tooltip for sighted mouse users.

Reply:
So if I understand well we are missing this part?

> Provide a text alternative inside a (or similar) element. Also include appropriate CSS to visually hide the element while keeping it accessible to assisitive technologies.

Reply:
My understanding is that if we don't add this alternative text, screen readers will not see anything to indicate that the user is an instructor and they will therefore have less information than other users which is not what we want.

Reply:
I understand that if we want to do the top solution yes but it's out of context IMO, it's too much for too little.
But by adding the title we already provide a tooltip for sighted mouse users which is a good first step.

Reply:
The current implementation is not accessible unless the icon is purely decorative and conveys 0 meaning.

If there's a meaning we need to add a visually hidden (`sr-only` or whatever the helper class is called in Ashley) span with the same text as the one in the `<i>` title.

I think font-awesome themselves recommend this approach for non interactive icons that are supposed to carry meaning.

Reply:
yes they do indeed as mentioned above, I doubt though the need of this in this context but I will do so :-)

Reply:
I did not work on Ashley so I'm not as knowledgeable as you about the context here.

I think in general though we should work from the assumption that if any user could see this UI element, then it needs to be accessible. Apart from my personal opinion, it's also necessary if we aim to be RGAA compliant and hopefully pass an audit later on.

Reply:
I think you are absolutely right and I got lazy for no reason on this one ! thank you for you feedback.